src/guidescanpy/commands/add_organism.py

```diff
@@ -11,17 +12,34 @@
 
 def insert_chromosome(organism, chr2acc_file):
     result = []
-    with open(
-        chr2acc_file,
-    ) as f:
-        reader = csv.DictReader(f, delimiter="\t")
+    with open(chr2acc_file) as infile, open("temp", "w", newline="") as outfile:
+        reader = csv.DictReader(infile, delimiter="\t")
+        new_header = [
+            field.lower().replace("# ", "").replace("#", "")
```

Comment:
I think we don't care about (and in fact shouldn't) replace all `#`s - only the ones at the start of the line (and then see if we can parse the line as the header, unless we have done so already).